StreamTemp-Integ.py

```diff
@@ -68,13 +64,17 @@
 
         # Define root directory of database and output
         # Modify this based on your own location
-        rootDatabase = os.path.join(os.path.sep, absRoot, 'scratch', 'SNTemp')  # CAMELS dataset root directory: /scratch/Camels
-        rootOut = os.path.join(os.path.sep, absRoot, 'TempDemo', 'FirstRun')  # Model output root directory: /data/rnnStreamflow
-
-        forcing_path = os.path.join(os.path.sep, rootDatabase, 'Forcing', 'Forcing_new', f_list)  # obs_18basins
-        forcing_data =[]#pd.read_feather(forcing_path)
-        attr_path = os.path.join(os.path.sep, rootDatabase, 'Forcing', 'attr_new', a_list)
-        attr_data =[]#pd.read_feather(attr_path)
+        rootDatabase = os.path.join(os.path.sep, absRoot, 'in')  # CAMELS dataset root directory: /scratch/Camels
+        rootOut = os.path.join(os.path.sep, absRoot, 'output')  # Model output root directory: /data/rnnStreamflow
```

Comment:
These are nice directory simplifications in these surrounding lines. Getting nitpicky, but `in` and `out`, or `input` and `output`, would make more sense to me than `in` and `output` as a pair of folders.